File: aula-04-eventos/Exercicio01/app/src/main/java/com/example/exercicio01/MainActivity.java
package com.example.exercicio01;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);


        EditText v1 = findViewById(R.id.editTextValue1);
        EditText v2 = findViewById(R.id.editTextValue2);
        TextView r = findViewById(R.id.textViewResult);


        Button btnSum = findViewById(R.id.buttonSum);
        btnSum.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                r.setText("Resultado: " + (Float.parseFloat(v1.getText().toString()) + Float.parseFloat(v2.getText().toString())));
            }
        });

        Button btnMinus = findViewById(R.id.buttonMinus);
        btnMinus.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                r.setText("Resultado: " + (Float.parseFloat(v1.getText().toString()) - Float.parseFloat(v2.getText().toString())));
            }
        });

        Button btnMultiplier =  findViewById(R.id.buttonMultiplier);
        btnMultiplier.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                r.setText("Resultado: " + (Float.parseFloat(v1.getText().toString()) * Float.parseFloat(v2.getText().toString())));
            }
        });

        Button btnDivider = findViewById(R.id.buttonDivider);
        btnDivider.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                r.setText("Resultado: " + (Float.parseFloat(v1.getText().toString()) / Float.parseFloat(v2.getText().toString())));
            }
        });
    }
}

File: aula-06-kotlin/Exercicio01/app/src/main/java/com/example/exercicio01/MainActivity.kt
package com.example.exercicio01

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.exercicio01.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        val v1 = binding.editTextValue1
        val v2 = binding.editTextValue2

        val btnSum = binding.buttonSum
        btnSum.setOnClickListener {
            val resultado = v1.text.toString().toFloat() + v2.text.toString().toFloat()
            binding.textViewResult.text = getString(R.string.resultado, resultado.toString())
        }

        val btnMinus = binding.buttonMinus
        btnMinus.setOnClickListener { val resultado = v1.text.toString().toFloat() - v2.text.toString().toFloat()
            binding.textViewResult.text = getString(R.string.resultado, resultado.toString()) }

        val btnMultiplier = binding.buttonMultiplier
        btnMultiplier.setOnClickListener {val resultado = v1.text.toString().toFloat() * v2.text.toString().toFloat()
            binding.textViewResult.text = getString(R.string.resultado, resultado.toString())}

        val btnDivider = binding.buttonDivider
        btnDivider.setOnClickListener { val resultado = v1.text.toString().toFloat() / v2.text.toString().toFloat()
            binding.textViewResult.text = getString(R.string.resultado, resultado.toString()) }
    }
}